### ISSUE TITLE
SVT: [Che] Error receiving profiling data

### ISSUE THREAD

**Issue by sujeilyfonseca** [eclipse/codewind#2220]
**Codewind version:** 0.9.0
**OS:** Red Hat Enterprise Linux 7 (64 bit)

**Che version:** 7.5.1
**Kubernetes cluster:** OKD/Openshift 

**Description:** In previous versions, users didn't receive any errors on the screen when running load tests to obtain profiling data for the Metrics Dashboard. However, in Codewind 0.9.0, users get an "Error receiving profiling data" every time they execute a load test. 

On one occasion, I didn't get anything in the Flame Graph, but then I couldn't reproduce that scenario. I can reproduce the error on the screen consistently.

_Codewind 0.8.1:_
<img width="1623" alt="Screen Shot 2020-02-14 at 8 07 25 AM" src="https://user-images.githubusercontent.com/49531960/74534120-11c66e00-4f01-11ea-8780-01f4bebad32b.png">
<img width="1618" alt="Screen Shot 2020-02-14 at 8 06 42 AM" src="https://user-images.githubusercontent.com/49531960/74534128-14c15e80-4f01-11ea-9e1e-9fadf0c6fa6e.png">
<img width="1619" alt="Screen Shot 2020-02-14 at 8 06 26 AM" src="https://user-images.githubusercontent.com/49531960/74534139-18ed7c00-4f01-11ea-917a-33958067014a.png">

_Codewind 0.9.0:_
<img width="1524" alt="Screen Shot 2020-02-14 at 7 42 51 AM" src="https://user-images.githubusercontent.com/49531960/74533123-dc208580-4efe-11ea-8dca-aefe5ddf185f.png">
<img width="1520" alt="Screen Shot 2020-02-14 at 7 43 12 AM" src="https://user-images.githubusercontent.com/49531960/74533135-e0e53980-4efe-11ea-9e6a-3327d1f8cfca.png">
<img width="1524" alt="Screen Shot 2020-02-14 at 7 43 26 AM" src="https://user-images.githubusercontent.com/49531960/74533142-e3479380-4efe-11ea-99d3-2ff366ccd0fa.png">

@jagraj

**Comment by tetchel** [comment]
I know that the new profiling code doesn't work on Che. I can quickly disable the load runner socket listener on Che, but i'm not sure if that's what we want to do, or if we can actually fix it. 

@ebuckle @tobespc

**Comment by tetchel** [comment]
confirmed that we can disable this feature for 0.9

once https://github.com/eclipse/codewind/pull/2123 is merged into master it should work there. you will continue to see the errors on master until then

**Comment by tetchel** [comment]
that message should be gone in 0.9 now

**Comment by sujeilyfonseca** [comment]
I no longer see the error message in 0.9.0. I'll close this issue once #2123 is merged to master.

**Comment by ebuckle** [comment]
#2123 Has now been merged in to master

**Comment by malincoln** [comment]
@sujeilyfonseca 2123 is now merged.  Can you close it now? Thanks

**Comment by sujeilyfonseca** [comment]
I verified this issue, and I no longer reproduce this problem.
